pyeudiw/satosa/default/response_handler.py

```diff
@@ -41,6 +41,9 @@
 from pyeudiw.trust.exceptions import InvalidJwkMetadataException
 from pyeudiw.jwt.exceptions import JWSVerificationError
 from pyeudiw.sd_jwt.exceptions import UnsupportedSdAlg, InvalidKeyBinding
+from pyeudiw.sd_jwt.schema import is_sd_jwt_kb_format
```

Comment:
it is not required anymore, since the mapping uses presentation_submission and or in the future duckle